fix package.json

[test](https://github.com/PrimeDeviation/mentus/actions/runs/9358743758/job/25761144833?pr=108#logs)
failed 39 minutes ago in 7s
Search logs
0s
1s
2s
0s
Run npm install
npm ERR! code EJSONPARSE
npm ERR! file /home/runner/work/mentus/mentus/package.json
npm ERR! JSON.parse Failed to parse json
npm ERR! JSON.parse Unexpected string in JSON at position 270 while parsing '{
npm ERR! JSON.parse   "name": "mentus",
npm ERR! JSON.parse   "version": "[1](https://github.com/PrimeDeviation/mentus/actions/runs/9358743758/job/25761144833?pr=108#step:4:1).0.'
npm ERR! JSON.parse Failed to parse package.json data.
npm ERR! JSON.parse package.json must be actual JSON, not just JavaScript.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/runner/.npm/_logs/202[4](https://github.com/PrimeDeviation/mentus/actions/runs/9358743758/job/25761144833?pr=108#step:4:5)-06-03T23_39_40_358Z-debug.log
Error: Process completed with exit code 1.

be sure to align this change with the manifest.json, and the other files in the project. Double check file paths for correct references

--- a/package.json
+++ b/package.json
@@ -7,7 +7,7 @@
     "build": "echo 'Building the project...'",
     "start": "echo 'Starting the project...'",
     "test": "echo 'Running tests...'"
-    }
+    },
   "license": "MIT",
   "dependencies": {},
   "devDependencies": {}

--- a/validate_json.py
+++ b/validate_json.py
@@ -1,0 +1,8 @@
+import json
+
+with open('package.json', 'r') as file:
+    try:
+        data = json.load(file)
+        print("JSON is valid.")
+    except json.JSONDecodeError as e:
+        print(f"JSON is invalid: {e}")

